src/app/models/markdown_content.rb

```diff
@@ -0,0 +1,2 @@
+class MarkdownContent < ActiveRecord::Base
```

Comment:
Do we want to `validates :slug, presence: true` instead of only relying on the db constraint?

Reply:
Done